### DISCUSSION TITLE
Dashboarding SQL Query

### DISCUSSION THREAD

**Discussion by SimonGitter** [fastapi/fastapi#2757] [Questions]
Hello, 

what would be the best practice if i would want to expose some data from different SQL Servers (Queries, Views).
I dont need to create tables or input validation, i just want to parse the username to the query/view and return the data. 
Does it even make sense to use FastApi for this? 

Thanks

**Comment by tiangolo** [comment]
Hmm, not sure I understand what you mean. Maybe it would be better to ask in an issue following the template, that will guide you to structure the question and build a minimal, self-contained example with what you need or what you have in mind. That will help me/us help you better.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.

**Reply by iudeen** [reply]
I'm sorry if I assumed something wrong.

You may very well use FastAPI to act as an API layer to create dashboards on your SQL data. But mind that, you'll also need a UI to be developed.

If you're looking for something really quick to visualize your data in DB, a tool like Grafana or PowerBI is what you need.
